### DISCUSSION TITLE
Tools for recording our own voice samples

### DISCUSSION THREAD

**Discussion by VigneshBaskar** [coqui-ai/TTS#1169] [General Q&A]
Dear Coqui community,
I am trying to record my own voice samples and currently I am using Audacity to record voice samples. Going back and forth between the text sentences in the Excel and Audacity is really cumbersome. Any recommendation of tools to efficiently record my own voice samples for TTS is highly appreciated please. Thank you

**Comment by thorstenMueller** [comment]
I've used and can recommend using [Mycroft Mimic-Recording-Studio](https://github.com/MycroftAI/mimic-recording-studio). If helpful you can watch [my video on Youtube](https://youtu.be/4YT8WZT_x48?t=259) on how to use it.

**Reply by VigneshBaskar** [reply]
Thank you very much @thorstenMueller. Just our of curiosity, is there any fully hosted online recording tool that you are aware of?

**Reply by thorstenMueller** [reply]
I directly found Mimic-Recording-Studio and didn't look any further. So, sorry no ;-).
